test/rekt/features/broker/data_plane.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"context"
+
+	"knative.dev/reconciler-test/pkg/feature"
+	"knative.dev/reconciler-test/pkg/state"
+)
+
+func DataPlaneConformance(brokerName string) *feature.FeatureSet {
+	return &feature.FeatureSet{
+		Name: "Knative Broker Specification - Data Plane",
+		Features: []feature.Feature{
+			*DataPlaneIngress(brokerName),
+			*DataPlaneDelivery(brokerName),
+			*DataPlaneObservability(brokerName),
+		},
+	}
+}
+
+func DataPlaneIngress(brokerName string) *feature.Feature {
+	f := feature.NewFeatureNamed("Ingress")
```

Comment:
Why not just using `NewFeature`? The name `DataPlaneIngress` should be fine

Reply:
I am not sure I am a fan of the test name reflection stuff. I like this to be explicit so I can filter on the string downstream